MacrobenchmarkSample/macrobenchmark/src/main/AndroidManifest.xml

```diff
@@ -15,10 +15,21 @@
   -->
 
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
     package="com.example.macrobenchmark">
 
+    <!--
+        Legacy external storage + storage permission used so CI can write outputs to e.g. /sdcard/.
+        This should ideally only required on the androidTest, but Samsung devices seem to expect
```

Comment:
Device doesn't affect how the manifest is produced, that comes from the single apk that's produced. Either the flag is in the output manifest or not-

Verify that putting it in androidTests works with Build > Inspect APK

May be that the Samsung device requires it, but pixel doesn't, but we should understand that clearly